src/assets/sass/utils/_global.scss

```diff
@@ -571,6 +569,16 @@ h3 {
   p {
     text-align: center !important;
   }
+}
+
+.horizontal-shuffle-answer,
+.pair-match-play-option,
+.missing-word-choice,
+.vertical-shuffle-choice,
+.short-answer-input,
+.pair-match-play-choice,
+.pair-match-image-choice,
+.choose-choice {
```

Comment:
@farmanahmed2007 why did you move it to global.scss could you move it to investigation build page or play page. it is hard to tell if this is related to play or build.